mindmaps-test/src/test/java/io/mindmaps/test/graql/reasoner/ReasonerTest.java

```diff
@@ -228,6 +229,22 @@ public void testNoRelationTypeWithRoles2(){
     }
 
     @Test
+    @Ignore
```

Comment:
You should put a `//TODO` here specifying to fix the test and maybe why you had to ignore it.